openeogeotrellis/configparams.py

```diff
@@ -28,6 +28,13 @@ def __init__(self, env=os.environ):
             "epod-master1.vgt.vito.be:6668,epod-master2.vgt.vito.be:6668,epod-master3.vgt.vito.be:6668",
         )
 
+        self.yarn_rest_api_base_url = env.get(
+            "YARN_REST_API_BASE_URL", "https://epod-master1.vgt.vito.be:8090"
+        )
+        self.yarn_auth_use_kerberos = smart_bool(
```

Comment:
Do we need this config param? AFAIK we always require Kerberos authentication to access YARN.